sites/kit.svelte.dev/src/routes/docs/[slug].svelte

```diff
@@ -0,0 +1,60 @@
+<script context="module">
+	export const prerender = true;
+
+	// TODO should use a shadow endpoint instead, need to fix a bug first
+	/** @type {import('@sveltejs/kit').Load} */
+	export async function load({ fetch, params }) {
+		const res = await fetch(`/docs/${params.slug}.json`);
+		const { section } = await res.json();
+
+		return {
+			props: {
+				section
+			}
+		};
+	}
+</script>
+
+<script>
+	import { Icon } from '@sveltejs/site-kit';
+	import '@sveltejs/site-kit/code.css';
+	import './_/docs.css';
+
+	export let section;
+</script>
+
+<svelte:head>
+	<title>{section.title} • Docs • SvelteKit</title>
+
+	<meta name="twitter:title" content="SvelteKit docs" />
+	<meta name="twitter:description" content="{section.title} • SvelteKit documentation" />
+	<meta name="Description" content="{section.title} • SvelteKit documentation" />
+</svelte:head>
+
+<div class="content listify">
+	<h1>{section.title}</h1>
+
+	<a href="https://github.com/sveltejs/kit/edit/master/documentation/docs/{section.file}">
+		<Icon size={50} name="edit" /> Edit this page on GitHub
+	</a>
```

Comment:
fixed — was getting hidden by the subsequent header's padding/negative margin combo (which exists so that the header doesn't stick to the very top of the page behind the nav bar when you click a `#` link)